test/indexer/test_hca_indexer.py

```diff
@@ -1256,6 +1259,45 @@ def test_accessions_fields(self):
             self.assertEqual(['E-AAAA-00'], project['array_express_accessions'])
             self.assertEqual(['PRJNA000000'], project['insdc_study_accessions'])
 
+    def test_cell_counts(self):
+        """
+        Verify the cell counts found in project, cell_suspension, and file entities
+        """
+        # Bundles from the canned staging area, both for project 90bf705c
+        # https://github.com/HumanCellAtlas/schema-test-data/
+        bundle_fqid = self.bundle_fqid(uuid='4da04038-adab-59a9-b6c4-3a61242cc972',
+                                       version='2021-01-01T00:00:00.000000Z')
+        self._index_canned_bundle(bundle_fqid)
+        bundle_fqid = self.bundle_fqid(uuid='d7b8cbff-aee9-5a05-a4a1-d8f4e720aee7',
+                                       version='2021-01-01T00:00:00.000000Z')
+        self._index_canned_bundle(bundle_fqid)
+        hits = self._get_all_hits()
+        field_paths = [
+            ('projects', 'estimated_cell_count'),
+            ('cell_suspensions', 'total_estimated_cells'),
+            ('files', 'matrix_cell_count')
+        ]
+        actual_values = NestedDict(2, int)
+        for hit in sorted(hits, key=lambda d: d['_id']):
+            entity_type, aggregate = self._parse_index_name(hit)
+            contents = hit['_source']['contents']
+            for inner_entity_type, field_name in field_paths:
+                for inner_entity in contents[inner_entity_type]:
+                    value = inner_entity[field_name]
+                    actual_values[aggregate][entity_type][inner_entity_type] += value
+        for aggregate in True, False:
+            for transformer in self.metadata_plugin.transformers():
+                entity_type = transformer.entity_type()
+                with self.subTest(aggregate=aggregate, entity_type=entity_type):
+                    expected_counts = {
+                        'projects': 10000 if aggregate and entity_type == 'projects' else 20000,
```

Comment:
Why is `estimated_cell_count` lower for a project aggregate than for a project contribution? If anything, shouldn't it be the same or higher?

Reply:
The two bundles in this test are for the same project, and both have a `Project.estimated_cell_count` of `10,000`.
Since the fingerprint `actual` in this test is a sum of cell counts (`actual[aggregate][entity_type][inner_entity_type] += value`), the fingerprint of the aggregate is `10,000` from two project contributions aggregated  with a `MaxAccumulator`, and the fingerprint from the contributions is `20,000` (`10,000` x2 )